src/spy/test.cljc

```diff
@@ -16,47 +16,89 @@
   ;; expected: (\"baz\")
   ;;   actual: [(\"foo\" \"bar\")]
   (is (spy/called-with? f \"baz\"))"
-  #?(:cljs (:require-macros [spy.test.cljs])
-     :clj  (:require [clojure.test :as t])))
+  #?(:clj (:require [clojure.test :as t])))
 
-(defn called-n [n] n)
-(defn called-once [] 1)
-(defn called-args [& args] args)
-(defn not-called [] 0)
+(defn expected-called-n-times
+  [n]
+  n)
```

Comment:
These should probably be improved to be more like the messages from spy.assert-messages.